lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -259,7 +259,7 @@ namespace :foreman_tasks do
 
         File.open(File.join(tmp_dir, 'index.html'), 'w') { |file| file.write(PageHelper.pagify(PageHelper.generate_index(tasks))) }
 
-        sh("tar cvzf #{export_filename} #{tmp_dir} > /dev/null")
+        system("tar", "czf", "#{export_filename}", "#{tmp_dir}")
```

Comment:
Now you can drop the string interpolation as it serves no purpose and do `system("tar", "czf", export_filename, tmp_dir)`

Reply:
Thanks for the suggestion and sorry for the delay. Changed as suggested.